Helper/Cacher.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Reach\Payment\Helper;
+
+//use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Cache\Type\FrontendPool;
+use Magento\Framework\Cache\Frontend\Decorator\TagScope;
+
+use Magento\Framework\Serialize\SerializerInterface;
+
+
+/**
+ * Helper Class
+ */
+//class Cacher extends AbstractHelper
+class Cacher extends TagScope
```

Comment:
If this class is not a subclass of `TagScope` then we get error when we try to compile code for dependency injection and interceptor generator as well as while running cache cleanup or flushing options.

Reply:
would it make sense to leave this note as a comment in the code? I wouldn't have guessed that a caching helper class would need to extend something called "TagScope"

Reply:
Done. Commit fcd15604f707f0a182d8bfbdf276f33dd0097cca 

Thanks Lizzy!